packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -463,11 +445,19 @@ const addTransition = async () => {
 
 <style scoped>
 main {
-	border: 1px solid var(--surface-border-light);
-	border-radius: var(--border-radius);
 	overflow: auto;
 }
 
+.accordion-group {
```

Comment:
I think you can just do `.p-accordion` the component has this class by default

Reply:
Thanks for the suggestion, just updated!